app/controllers/search_goods.js

```diff
@@ -0,0 +1,92 @@
+import Ember from "ember";
+import searchModule from "./search_module";
+import _ from "lodash";
+
+export default searchModule.extend({
```

Comment:
I'm assuming you're using @abulsayyad123 's infinite-list

If that's the case, there's no need to inherit from `search_module`. We also want to avoid that as much as possible (we want to strip out that nightmare class, and eventually delete it altogether).

Reply:
yeah sure, inherited it from ember component now.